src/components/LoginComponent.js

```diff
@@ -31,11 +31,12 @@ function LoginComponent(props){
         
         authenticate(email, password).then(
             data => {
+                localStorage.setItem("email",email);
+                localStorage.setItem("password",password);
                 console.log("logged in!",data);
+                localStorage.setItem("firstname",data.idToken.payload["custom:firstname"]);
+                localStorage.setItem("lastname",data.idToken.payload["custom:lastname"]);
                 var user_id=data.accessToken.payload.username;
-                console.log(user_id);
```

Comment:
Change as per this example - https://github.com/jeenathampi/dropbox-client/blob/master/src/components/Dashboard.js#L21